Assets/Scripts/WorldEngine/World.cs

```diff
@@ -2674,35 +2683,46 @@ public void FinalizeTerrainAlterationDrainageRegen()
 
     public bool AddToDrainageRegen(TerrainCell cell, bool resetDrainage = true, bool resetTerrain = true)
```

Comment:
I'm not really sure what this does? Including a summary would be nice.

Reply:
Added a summary to this function. I'll keep adding summaries to functions I modify whenever I remember to do so. Please keep reminding me if I forget to do so.